website/docs/docs/core/connect-data-platform/extrica-setup.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Extrica Setup"
+description: "Read this guide to learn about the Extrica Trino Query Engine setup in dbt."
+id: "extrica-setup"
+meta:
+  maintained_by: Extrica, Trianz 
+  authors: Gaurav Mittal, Viney Kumar, Mohammed Feroz, and Mrinal Mayank
+  github_repo: 'extricatrianz/dbt-extrica'
+  pypi_package: 'dbt-extrica'
+  min_core_version: 'v1.7.2'
+  cloud_support: 'Not Supported'
+  min_supported_version: 'n/a'
+  platform_name: 'Extrica'
+---
+<h2> Overview of {frontMatter.meta.pypi_package} </h2>
+
+<ul>
+    <li><strong>Maintained by</strong>: {frontMatter.meta.maintained_by}</li>
+    <li><strong>Authors</strong>: {frontMatter.meta.authors}</li>
+    <li><strong>GitHub repo</strong>: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></li>
+    <li><strong>PyPI package</strong>: <code>{frontMatter.meta.pypi_package}</code> <a href={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}`}><img src={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}.svg`}/></a></li>
+    <li><strong>Supported dbt Core version</strong>: {frontMatter.meta.min_core_version} and newer</li>
+    <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
+    <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
+    </ul>
+<h2> Installing {frontMatter.meta.pypi_package} </h2>
+
+Use `pip` to install the adapter, which automatically installs `dbt-core` and any additional dependencies. Use the following command for installation:
+
+<code>python -m pip install {frontMatter.meta.pypi_package}</code>
+
+<h2> Configuring {frontMatter.meta.pypi_package} </h2>
+
+<p>For {frontMatter.meta.platform_name}-specifc configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
```

Comment:
Since there's not Extrica-specific configuration, this little section should be removed.

```suggestion
```